kore/src/Kore/Repl/Parser.hs

```diff
@@ -275,9 +275,8 @@ logScope =
             <* optional (literal ",")
 
 logType :: Parser LogType
-logType = noLogging <|> logStdOut <|> logFile
+logType = logStdOut <|> logFile
```

Comment:
I did't remove `NoLogging` constructor from `data LogType` because I thought it might be used in other cases beside command line arguments, but only removed the possibility for it to be specified by user. I don't know if this is correct

Reply:
That's fine, but it should be `stderr` again instead of `stdout`.